### DISCUSSION TITLE
Is there a way to use filament fields separately ? as a UI Library

### DISCUSSION THREAD

**Discussion by mybouhssina** [filamentphp/filament#11068] [Help] [answered]
### Package

Form builder

### Package Version

3

### How can we help you?

Hello,
Is there a way to use filament fields separately ? like a UI Library.
Not forms in Livewire components [as described here](https://filamentphp.com/docs/3.x/forms/adding-a-form-to-a-livewire-component).
Thank you,

**Comment by leandrocfe** [comment, marked as answer]
this?
https://filamentphp.com/docs/3.x/support/blade-components/overview

**Comment by mybouhssina** [comment]
Exactly, thank you !
